src/api/views.py

```diff
@@ -30,7 +30,8 @@
     ),
     "invalid": "Enter a domain using only letters, numbers, or hyphens (though we don't recommend using hyphens).",
     "success": "That domain is available!",
-    "error": "Error finding domain availability.",
+    "error": "Error finding domain availability. Please wait a few minutes and try again.\
```

Comment:
**blocking change**: The slack thread with Katherine mentioned using our already-designer approved Registry connection error message (a few lines up from the message I sent you earlier). It is in registrar/utility/errors.py and it is the one associated with Cannot_contact_registry. If you can reference that one directly that would be ideal, otherwise I would copy and paste it here instead.

Reply:
That being said, @Katherine-Osos would erin's suggestion for a registry connection error above work as a replacement or would you prefer the one already suggested in my message "We’re experiencing a system connection error. Please wait a few minutes
and try again. If you continue to receive this error after a few tries,
contact help@get.gov"

Reply:
Erin's suggestion works!

Reply:
I was going to say Erin's suggestion works ... but I realized this same message should appear when searching for a domain in the request flow (right?). In that case, we should offer the contact email, so let's use the existing message we use elsewhere: 

"We’re experiencing a system connection error. Please wait a few minutes and try again. If you continue to receive this error after a few tries, contact [help@get.gov](mailto:help@get.gov)"

Reply:
Updated the error message to make the error message to be the same as the generic registry error by pulling from the constants set in our GenericError class - the only concern I have is that it does this by referencing a helper function _error_mapping in GenericError and I was wondering if this was good practice. It's not exactly a private function and I was wondering if we should make any code format changes that would be good for this @abroddrick

Reply:
I totally see what you're saying, normally you wouldn't directly access something with a leading _. I would say just add a function in GenericError  getRegistryConnection error and have it just be one line that has a return with the _error_mapping[...]. That would probably be a bit easier to read anyways

Reply:
Sounds good! Just updated to make a class function that gives the correct error message for the corresponding code